wrappers/Gateway.ts

```diff
@@ -10,9 +10,60 @@ import {
     SendMode,
     toNano,
     TupleItemInt,
+    Slice,
 } from '@ton/core';
-import { evmAddressToSlice, loadHexStringFromBuffer, signCellECDSA } from '../tests/utils';
-import { Wallet as EVMWallet } from 'ethers'; // copied from `gateway.fc`
+import { Wallet } from 'ethers'; // copied from `gateway.fc`
```

Comment:
Let's keep `EVMWallet` as a  term because by default we assume that the wallet is the TON wallet

Reply:
`// copied from gateway.fc` - should we delete this?

Reply:
88f6d74